### DISCUSSION TITLE
Toggle focus on replace field

### DISCUSSION THREAD

**Discussion by andrealeone** [zed-industries/zed#8144] [Feature Requests]
G'day folks,

this is my first contribution to the repo, so allow me a few words of appreciation for all your work. I loved Zed so far, its UI, and the philosophy that drives its development. I'm looking forward to making it my default IDE.

There is one little thing that is missing in my everyday workflow: a key binding to focus the replace field in the buffer search. Usually, I move my cursor on a word, hit <kbd>⌘</kbd> + <kbd>F</kbd> to have it in the find field, then move to the replace field and hit <kbd>Enter</kbd>.  

I know I can use <kbd>⌘</kbd> + <kbd>⇧</kbd> + <kbd>H</kbd> to toggle the replace field triggering the `search::ToggleReplace` action, but is there an action to focus it? Something like `project_search::ToggleFocus` but for the replace field?

I'd love to help.

**Comment by Moshyfawn** [comment]
If I understand [the code correctly,](https://github.com/zed-industries/zed/blob/f19ab464c7b0610bc1479195ca86572c8205d700/crates/search/src/project_search.rs#L1458-L1471) there's no such alternative function in Zed at the moment, which might mean that it needs a feature request.
